fineract-provider/src/main/java/org/apache/fineract/infrastructure/bulkimport/importhandler/saleofinsuranceorassistance/SaleOfInsuranceOrAssistanceImportHandler.java

```diff
@@ -83,7 +82,7 @@ private ClientBuyProcessData readInsuranceSaleData(final Workbook workbook, fina
         data.setChannelHash(channelHash);
         data.setRowIndex(row.getRowNum());
         data.setSaleOfInsuranceOrAssistance(true);
-        data.setDateFormat(dateFormat);
+        data.setDateFormat("dd/MM/yyyy");
```

Comment:
was this intentional ?

Reply:
Yes. Because when you change the language and not the dateformat from front end then exception occurs when getting date from json element. Since this is only for bulk import of insurance sales so it is fine to hardcode here

Reply:
If you see below, i am removing the date from json object and adding back with the same hardcoded date format